pdp/pdpfw.mac

```diff
@@ -214,37 +233,25 @@ doneprntn:
 	CALL TNYDLY
 
 	jmp @#173000 ; Reboot!
-	
-end: 
-	wait
-	br end
 
 cartTitle:
 	.ascii "  Genjitsu"
 	.byte 16 ; 16o start russ
-	.ascii " smp 0.2  "
+	.ascii " smp 0.3  "
 	.byte 17 ; 17o end russ
 	.byte 0
 cartMoar:
-	.ascii "^"
+	.ascii " [BK]"
 	.byte 15, 0
 cartCrlf:
 	.byte 12,15,0
 cartMsgFilename:
 	.byte 16
-	.ascii "iMQ SU]ESTWU@]EGO / NOWOGO FAJLA?"
-	.byte 17,12,15,0
+	.ascii "iMQ FAJLA (SU]/NOW)?"
+	.byte 17,0
 cartRst:
 	.byte 16
 	.ascii "pEREZAGRUVA@SX!"
 	.byte 17,0
 
-	.= 1000
-
-	; -------------------- 
-	; !! END OF BOOTLOADER !!
-	;  Further data after 1000o  would need to be read manually !
-	; --------------------
-
```

Comment:
This needs to remain, otherwise if the code above extends beyond 1000o the compiler will not produce an error and you will end up with a bunch of headache and fun time figuring out why is that so :P